src/AbstractFixer.php

```diff
@@ -116,6 +116,10 @@ public function configure(array $configuration = null)
         }
 
         if (null === $configuration) {
+            if (getenv('PHP_CS_FIXER_FUTURE_MODE')) {
+                throw new \InvalidArgumentException('Parameter must not be `null`.');
```

Comment:
this check looks off, `getenv` will retrun `false|string` so never `null` if IIRC?